deployments/lookup_hints.yaml

```diff
@@ -2,31 +2,38 @@ status: Converted
 registry_authors:
 - Nicolas Berrios
 - Jack Fitzsimons
+- James Honaker
 tier: 1
 deployment:
   name: Lookup Hints
   data_curator: Apple
   intended_use: Web Browsing
   data_product_type: Summary statistics # TODO: Is this correct?
   data_product_region: Not specified
-  description: For Lookup Hints, Apple uses a privacy budget with epsilon
-    of 4, and limits user contributions to two donations per day.
-  publication_date: '1999-01-01' # TODO: Is this correct?
+  description: 'The Apple differential privacy implementation incorporates the concept of a per-
+    donation privacy budget (quantified by the parameter epsilon), and sets a strict limit on
+    the number of contributions from a user in order to preserve their privacy... For Lookup Hints, Apple uses a privacy budget with epsilon
+    of 4, and limits user contributions to two donations per day.'
+  publication_date: '2016-01-01' # Not in source by referenced in Tang et al. https://arxiv.org/abs/1709.02753
   additional_information: https://www.apple.com/privacy/docs/Differential_Privacy_Overview.pdf
-  # dp_flavor: TODO
+  dp_flavor:
+    name: Pure DP
+    data_domain: '' # TODO
+    unprotected_quantities: '' # TODO
   privacy_loss:
     privacy_unit: User-level
-    # privacy_unit_description: TODO
+    # privacy_unit_description:
     privacy_parameters:
-      epsilon: 4.0
+      epsilon: 8.0
```

Comment:
Any idea how the value was confused? Wouldn't hurt to add a note to explain this if there is a question in the future.

Reply:
Indeed.  I've been attempting to add quotes to justify all the empty field additions and changes.  In this case, the quote is above:
description: |
    "The Apple differential privacy implementation incorporates the concept of a per-
    donation privacy budget (quantified by the parameter epsilon), and sets a strict limit on
    the number of contributions from a user in order to preserve their privacy... For Lookup Hints, Apple uses a privacy budget with epsilon
    of 4, and limits user contributions to two donations per day."